README.md

```diff
@@ -49,6 +49,58 @@ variations and defines a default region.
 
 The above query would also produce the same output.
 
+# Development
+
+For ease of use, instead of installing
+[pgxn-client](https://pgxn.github.io/pgxnclient/) locally, you can use the
+provided docker container.
+
+* Start the container (it will stay up for 10 days by default):
+
+```sh
+docker-compose up -d
+```
+
+* Enter the container:
+
+```sh
+docker-compose exec pgxn-tools bash
+```
+
+* Then you can enter the PostgreSQL instance:
+
+```sh
+psql -U postgres -d contrib_regression
```

Comment:
@madtibo Here the command didn't works because postgres is not launched and if I launch a postgres database using:

```
sudo pg-start 15
```

I get an error:

```sh
root@09558f0f20d3:/# psql -U postgres -d contrib_regression
psql: error: connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: FATAL:  database "contrib_regression" does not exist
```

Reply:
@Its-Alex   en effet, il manque des actions. J'ai ajouté la création de l'instance et de la base de données.